backend/controllers/listingsController.js

```diff
@@ -17,22 +17,25 @@ const createListing = async (req, res) => {
     address: "address",
     neededByDate: "neededByDate"
   };
-  const emptyFields = Object.entries(requiredFields)
-    .filter(([key, value]) => !req.body[key])
-    .map(([key, value]) => value);
-  if (emptyFields.length > 0) {
-    return res
-      .status(400)
-      .json({error: "Please fill in all fields", emptyFields})
-  };
-  try{
-      const listing = await Listing.create({organisationName, title, requirement, description, address, neededByDate});
-      res.status(200).json(listing);
-    } catch (error){
-        res.status(400).json({error: error.message});
-    }
+
+  // const emptyFields = Object.entries(requiredFields)
+  //   .filter(([key, value]) => !req.body[key])
+  //   .map(([key, value]) => value);
+
+  // if (emptyFields.length > 0) {
+  //   return res
+  //     .status(400)
+  //     .json({error: "Please fill in all fields", emptyFields})
+  // };
```

Comment:
this code?

Reply:
Yes we can remove the code as we have set up validation rules in model>listings